src/test/kotlin/lotto/domain/FakeLottoNumberGenerator.kt

```diff
@@ -0,0 +1,7 @@
+package lotto.domain
+
+class FakeLottoNumberGenerator : LottoNumberGenerator {
+    override fun generate(): List<Int> {
+        return listOf(1, 2, 3, 4, 5, 6)
+    }
+}
```

Comment:
```suggestion
class FakeLottoNumberGenerator( val list: List<Int> ) : LottoNumberGenerator {
    override fun generate(): List<Int> {
        return list
    }
}
```
처럼 재활용 가능한 구조로도 활용할수 있을거같아요!